macaroonbakery/utils.py

```diff
@@ -87,3 +100,44 @@ def visit_page_with_browser(visit_url):
     @param visit_url: where to prove your identity.
     '''
     webbrowser.open(visit_url, new=1)
+    print('Opening an authorization web page in your browser.')
+    print('If it does not open, please open this URL:\n', visit_url, '\n')
+
+
+def Cookie(
```

Comment:
functions should be named in lower case

Reply:
interesting really, because as far as the usage is concerned, this is a very similar usage to  http_cookiejar.Cookie. I'll rename it.

Reply:
done